app/components/container/containerController.js

```diff
@@ -116,9 +116,23 @@ function ($scope, $state, $stateParams, $filter, Container, ContainerCommit, Ima
     });
   };
 
-  $scope.remove = function () {
+  $scope.confirmRemove = function () {
+    if ($scope.container.State.Running) {
+      ModalService.confirmDeletion(
+        'You are about to delete a running container.',
```

Comment:
OK, this will change the modal for remove service and remove user too.
Do you want me to change the associated messages too ?
See : https://github.com/portainer/portainer/search?utf8=%E2%9C%93&q=confirmDeletion&type=